ONSdigital/eq-questionnaire-runner
# Remove census survey configurations from Runner

### What is the context of this PR?
Remove census survey configurations across runner in python files, json schemas and functional tests as they are no longer used. Anything that was `individual response` was left untouched. Within `confirmation-email-sent.html` the replacement email (line 25) is something that isn't known. I also changed `test_list_collector_list_summary.json`

### How to review
- Ensure the changes made are correct and if any files need to be updated
- Check if changes to `test_list_collector_list_summary.json` are appropriate or need to be reverted

### Checklist
